app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app import errors
```

Comment:
Use multiple imports:
`from app.errors import (NotVaccinatedError...)`

Reply:
Like an example in your checklist 
![image](https://user-images.githubusercontent.com/104218550/205485935-a316ee93-bf4e-40d1-8c08-03d7c46de9ba.png)